porting/common-system-build-errors.rst

```diff
@@ -41,8 +41,7 @@ Flex locale error
    flex-2.5.39: loadlocale.c:130: _nl_intern_locale_data: Assertion `cnt < (sizeof (_nl_value_type_LC_TIME) / sizeof (_nl_value_type_LC_TIME[0]))' failed.
    Aborted (core dumped)
 
+This seems to be a problem with locales and the prebuilt ``flex``. You can avoid this by using the flex installed on your host::
 
-
-Seems to be a problem with locales and the prebuilt `flex`. You can avoid this by using the "C" locale while building::
-
-   LANG=C make systemimage
+    export USE_HOST_LEX=yes
```

Comment:
Not sure what's the best approach here. LineageOS seems like it might use LC_ALL=C instead of USE_HOST_LEX=yes. This is also what I PR-ed in https://github.com/Halium/docs/pull/130

Either way, both seem to work for me and solve my issues with 7.1 build on 18.04.

Reply:
Do *both* work, or do *either* work?

Reply:
I tried both in independently with a clean build and both worked for me.